src/components/AddItemForm.js

```diff
@@ -16,23 +33,61 @@ const AddItemForm = () => {
     setNewItem(e.target.value);
   };
 
+  //Check for duplication:
+  //get the existing items list from firebase
+  //loop through the existing items list to check if there is a match with current item
+  //set itemExist to true if duplication and return it
+
+  const checkDuplication = (newItem) => {
+    let existingData = datas;
+
+    let itemExist = false;
+
+    existingData.forEach((object) => {
```

Comment:
can we use a more descriptive name instead of `object`? for example `itemObject`